_source/logzio_collections/_metrics-sources/azure-monitor.md

```diff
@@ -0,0 +1,179 @@
+---
+title: Ship metrics using Azure Monitor
+logo:
+  logofile: azure-monitor.svg
+  orientation: vertical
+data-source: Azure Monitor
+contributors:
+  - imnotashrimp
+  - yotamloe
+shipping-tags:
+  - azure
+---
+
+To monitor your Azure service metrics,
+we recommend configuring Metricbeat
+to collect metrics from Azure Monitor.
+Azure Monitor collects log and metric data
+from your Azure services.
+The Metricbeat configurations on this page
+collect metrics from Azure Monitor
+and forwards them to Logz.io.
+
+#### Metricbeat setup
+
+**Before you begin, you'll need**:
+[Metricbeat 7.6](https://www.elastic.co/guide/en/beats/metricbeat/7.6/metricbeat-installation.html),
+[Azure CLI 2.0](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest) or higher
+
+<div class="tasklist">
+
+##### Log in to Azure with the CLI
+
+Run this command in the command line:
+
+```shell
+az login
+```
+
+To complete the process, sign in through your browser.
+
+##### Get your subscription details
+
+Run this command:
+
+```shell
+az account show | grep '"id"\|"tenantId"'
+```
+
+The response shows the subscription ID (the `id` field)
+and tenant ID (the `tenantId` field).
+You'll need this information later on, so paste it in your text editor.
+
+###### Sample response
+
+```text
+"id": "d94b1fba-0289-557e-b063-0b6bfc1bdca0",
+"tenantId": "9ae0715a-0689-56e8-bb88-2b22f1fa7299",
+```
+
+##### Create a new Azure AD application
+
+Run this command to create an AD application with reader permissions.
+
+Replace `<<SUBSCRIPTION-ID>>` with the `id` value from step 2.
+
+```shell
+az ad sp create-for-rbac --role reader \
+--scopes /subscriptions/<<SUBSCRIPTION-ID>> \
+-n logzio-metricbeat \
+| grep '"appId"\|"password"'
+```
+
+The response shows the client ID (the `appId` field)
+and client secret (the `password` field).
+You'll need this information later on, so paste it in your text editor.
+
+###### Sample response
+
+```text
+"appId": "3dcdf53e-f93f-5902-8df2-235c8635aa4d",
+"password": "e6ab6d24-4907-5d11-a132-a171ef55355d",
+```
+
+##### Download the Logz.io certificate
+
+Now we'll set up your Metricbeat server.
```

Comment:
I'm not sure server is the right word

Reply:
Not sure I know of an alternate word here. It's the same language I use when you're setting up an independent Filebeat instance to route logs through.

Do you have a suggestion for a better word?

Reply:
can we just explain why we need the cert here?

Reply:
Yes, but it still needs some context as to _where_ we're putting it - i.e., on the metricbeat server

Reply:
why not just metricbeat

Reply:
Because you're not installing it in metricbeat? You're downloading it to a location on the server that hosts metricbeat.